src/jit/lsraarm.cpp

```diff
@@ -717,6 +717,11 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
             info->internalIntCount = 1;
             break;
 
+        case GT_PUTARG_SPLIT:
+            info->srcCount = 0;
+            info->dstCount = 0;
```

Comment:
How is the register modeled for this? Either this should produce multiple registers, that are then consumed by the call, OR the register fields should be contained, and consumed by the call. Either way, I think that this needs additional handling.